### DISCUSSION TITLE
website beta version

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#29] [General]
Hello
EHS implemented a new feature "beta version" which allows the user to see the new design of a page with the option to switch back to the old design using a toggle switch. The feature's purpose is mainly to get the users' feedback on the new design compared to the old one. The question is if it is compatible to add a highlight strip or coloring like the alert to alert the user that the user is browsing the beta version of a new design.
Please let us know your feedback

**Comment by moiz-conceptlz** [comment]
Hi @ahmedabbous 

Placing alert banners is absolutely at the discretion of the entity - there is no rule that says you may not add such a banner if required.

It is, however, my personal suggestion that multiple header or footer related banners on a single page - are never a good idea. For example, you must ask the question internally - is the alert really needed where it is? Should it be permanent? Should it exist on every page - or only where relevant? Does the content of the alert actually pertain to an alert?

There is also room for being creative. If the alert banner under the navigation must remain, then perhaps the fact the user is viewing a legacy version can be similar to the "dismissing banner example" on [https://designsystem.gov.ae/docs/components/banner](https://designsystem.gov.ae/docs/components/banner) where the objective is that is an overlayed banner placed on the bottom of the page. 

It is encouraged to be creative in design. Components exist to provide you with what is required. You may still add certain features as a temporary notification in a creative manner that you may find usable.
